doc/modules.rst

```diff
@@ -30,20 +30,37 @@ Registration
 A module must import ``mig/modules``.
 
 A module registers itself at runtime via its ``init()`` function which must
-call ``modules.Register`` with a module name and a "runner function" returning
-an anonymous object (``interface {}``) that must implement the
+call ``modules.Register`` with a module name and an object implementing
```

Comment:
go doesn't have objects. `modules.Moduler` is an interface. Could you replace `object` with `interface` in that sentence?